projects/js-packages/publicize-components/src/components/post-publish-share-status/share-status.tsx

```diff
@@ -6,15 +6,23 @@ import Notice from '../notice';
 import { ShareStatusModalTrigger } from '../share-status';
 import styles from './styles.module.scss';
 
+type ShareStatusProps = {
+	reShareTimestamp?: number | null;
```

Comment:
We don't need to set it to `null`, using `undefined` should work fine.

```suggestion
	reShareTimestamp?: number;
```

Reply:
Also, should we name that prop as something generic instead of `reShareTimestamp`? Like `lastTimestamp`, `countAfterTimestamp` or something similar.

The reason for that suggestion is not to tightly couple that component with reshare and it can be used elsewhere in future.

Reply:
Currently I think the only valid use-case to reuse this is for resharing, because for sharing the first time you won't have any previous shares, that's why I named it that, we can always rename it if a new scenario comes up.

Reply:
Fair enough. Thanks